pkg/cmd/server/api/types.go

```diff
@@ -1407,3 +1408,16 @@ type DefaultAdmissionConfig struct {
 	// Disable turns off an admission plugin that is enabled by default.
 	Disable bool
 }
+
+// TemplateServiceBrokerConfig holds information related to the template
+// service broker
+type TemplateServiceBrokerConfig struct {
+	// Enabled switches on the types, API and controllers associated with the
+	// template service broker
+	Enabled bool
+
+	// TemplateNamespaces indicates the namespace(s) in which the template service
+	// broker looks for templates to serve.  If empty, the default is the
```

Comment:
> and it's sane(not magic) because it's consistent with what the product has done from day one. ("openshift" has always been the default namespace where your templates are registered and found).

To my knowledge, that was never defaulted in the `policyConfig` which is used to grant the permission that make shared.